### DISCUSSION TITLE
Nested collapsible fieldsets can't be expanded

### DISCUSSION THREAD

**Discussion by aanckar** [sanity-io/sanity#3488] [Sanity Studio Support]
When nesting an object field with collapsible fieldsets inside an array field, the collapsed fieldsets can't be expanded (clicking doesn't do anything). Example schema [here](https://github.com/aanckar/sanity-studio-v3-test/blob/main/schemas/index.ts).

This issue appears eg. when using field level translations following the [recommended approach](https://www.sanity.io/docs/localization) inside an array field.

Steps to reproduce:
1. Use example repo ([https://github.com/aanckar/sanity-studio-v3-test](https://github.com/aanckar/sanity-studio-v3-test))
2. Launch studio, create a page
3. Add an item to the blocks field
4. Attempt to expand the translations fieldset

**Comment by wernerglinka** [comment]
It seems that `collapsible` doesn't work, regardless what the initial state is.
